python/cudf/cudf/tests/test_series.py

```diff
@@ -14,6 +14,9 @@
 import pytest
 
 import cudf
+from cudf.core.dtypes import Decimal32Dtype
+from cudf.core.dtypes import Decimal64Dtype
+
```

Comment:
```suggestion
```

You can get these from the `cudf` module

Reply:
Thank you for pointing that out! I've removed it as suggested.